Rakefile

```diff
@@ -1,38 +1,16 @@
+require 'packaging'
+
 RAKE_ROOT = File.expand_path(File.dirname(__FILE__))
 
-begin
-  load File.join(RAKE_ROOT, 'ext', 'packaging', 'packaging.rake')
-rescue LoadError
-end
+Pkg::Util::RakeUtils.load_packaging_tasks
 
-build_defs_file = File.join(RAKE_ROOT, 'ext', 'build_defaults.yaml')
-if File.exist?(build_defs_file)
-  begin
-    require 'yaml'
-    @build_defaults ||= YAML.load_file(build_defs_file)
-  rescue Exception => e
-    STDERR.puts "Unable to load yaml from #{build_defs_file}:"
-    raise e
-  end
-  @packaging_url  = @build_defaults['packaging_url']
-  @packaging_repo = @build_defaults['packaging_repo']
-  raise "Could not find packaging url in #{build_defs_file}" if @packaging_url.nil?
-  raise "Could not find packaging repo in #{build_defs_file}" if @packaging_repo.nil?
-
-  namespace :package do
+namespace :package do
  #   desc "Bootstrap packaging automation, e.g. clone into packaging repo"
```

Comment:
No longer seem relevant.

Reply:
@underscorgan do we need these?

Reply:
we need to keep the tasks, though the commented out `desc` can be removed. 
We need to keep the tasks since at this point our automation in many places will still try to call `package:bootstrap` and `package:implode` so it can function with projects that are using packaging from source instead of from a gem. We can go back and clean all of this up once all the things are using packaging as a gem, but this allows the migration to happen more slowly

Reply:
Ah right right, ok cool, updated the file!